docs/src/guide/built_in_collabs.md

```diff
@@ -8,24 +8,47 @@ For a type `X`, we use `C(X)` to denote a collaborative version of `X`. The tabl
-| `Map<K, V>`, `V` immutable                       | [`LWWCMap<K, V>`](../api/collabs/classes/LWWCMap.html)                    | [`CMap<K, V>`](../api/collabs/interfaces/CMap.html) implementations           |
-| `Map<K, V>`, `V` mutable                         | [`DeletingMutCMap<K, C(V)>`](../api/collabs/classes/DeletingMutCMap.html) | [`CMap<K, C(V)>`](../api/collabs/interfaces/CMap.html) implementations        |
-| `Array<T>`, `T` immutable                        | [`PrimitiveCList<T>`](../api/collabs/classes/PrimitiveCList.html)         | [`CList<T>`](../api/collabs/interfaces/CList.html) implementations            |
-| `Array<T>`, `T` mutable                          | [`DeletingMutCList<T>`](../api/collabs/classes/DeletingMutCList.html)     | [`CList<C(T)>`](../api/collabs/interfaces/CList.html) implementations         |
-| \*(Any immutable `T`) (as opaque value)          | [`LWWCVariable<T>`](../api/collabs/classes/LWWCVariable.html)             | [`CVariable<T>`](../api/collabs/interfaces/CVariable.html) implementations    |
-| \*(Any mutable `T`) (as opaque value)            | [`LWWMutCVariable<C(T)>`](../api/collabs/classes/LWWMutCVariable.html)    | [`CVariable<C(T)>`](../api/collabs/interfaces/CVariable.html) implementations |
-| \*`boolean`                                      | [`TrueWinsCBoolean`](../api/collabs/classes/TrueWinsCBoolean.html)        | [`CBoolean`](../api/collabs/interfaces/CBoolean.html) implementations         |
-| \*`number` (for counting or adding)              | [`CCounter`](../api/collabs/classes/CCounter.html)                        |
-| \*`number` (for counting, adding, and resetting) | [`ResettableCCounter`](../api/collabs/classes/ResettableCCounter.html)    |
-| \*`number` (for general arithmetic)              | [`CNumber`](../api/collabs/classes/CNumber.html)                          |
-| \*`string` (as text)                             | [`CText`](../api/collabs/classes/CText.html)                              |
-
-\*`X` denotes a _variable holding type `X`_, or alternatively a mutable wrapper object `{ value: X }`. E.g., `CCounter` is the collaborative version of an object that you can mutate in-place by calling e.g. `obj.add(3)` and that you can read by calling `obj.value`.
+| Ordinary type `X`                                | Collaborative version `C(X)`                                                                                                      |
+| ------------------------------------------------ | -------------------------------------------------------------------------  |
+| Any [immutable](#immutable-value-collections)   `T`     | [`LWWCVariable<T>`](../api/collabs/classes/LWWCVariable.html)            |
+| Any [mutable](#mutable-value-collections) `T`           | [`LWWMutCVariable<C(T)>`](../api/collabs/classes/LWWMutCVariable.html)     |
```

Comment:
For the variables, it might be confusing to link to the Immutable and Mutable sections below, since those only talk about collection of values, not individual variables.